v20.1/change-data-capture.md

```diff
@@ -540,17 +541,18 @@ In this example, you'll set up a changefeed for a single-node cluster that is co
     ~~~ shell
     $ ./bin/kafka-avro-console-consumer \
     --bootstrap-server=localhost:9092 \
-    --property print.key=true \
     --from-beginning \
     --topic=office_dogs
     ~~~
 
     ~~~ shell
-    {"id":{"long":1}}	{"after":{"office_dogs":{"id":{"long":1},"name":{"string":"Petee"}}}}
-    {"id":{"long":2}}	{"after":{"office_dogs":{"id":{"long":2},"name":{"string":"Carl"}}}}
+    {"after":{"office_dogs2":{"id":{"long":1},"name":{"string":"Petee H"}}}}
```

Comment:
Where's this office_dogs2 coming from? I see a CREATE CHANGEFEED FOR office_dogs2, but I don't see where we create that table

Reply:
Good catch - that's a typo from me running a second changefeed. It should be `office_dogs` still.